### PR TITLE
data manipulation

### PR DESCRIPTION
**Pull request by untzag**
recovers the following methods of data:
- `bring_to_front`
- `remove_channel`
- `rename_attrs` (changed to `rename`, method of `Group`)

### CODE REVIEW

File: WrightTools/data/_data.py [hunk 1]
```diff
@@ -1565,6 +1577,109 @@ def remove_channel(self, channel, verbose=True):
         if verbose:
             print('channel {0} removed'.format(name))
 
+    def remove_variable(self, variable, verbose=True):
```
Comment (ksunden) [inline WrightTools/data/_data.py]: Do we wish to have both singular and plural remove variable?
If so, this should just call the plural version with implied=False, I think, but leaning towards just the one function

File: WrightTools/data/_data.py [hunk 2]
```diff
@@ -1565,6 +1577,109 @@ def remove_channel(self, channel, verbose=True):
...
+            names[index] = v
+        self.variable_names = names
+        # update axes
+        new = self.axis_expressions
+        for i, v in enumerate(kwargs.keys()):
+            for j, n in enumerate(new):
+                new[j] = n.replace(v, '{%i}' % i)
+        for i, n in enumerate(new):
+            new[i] = n.format(*kwargs.values())
+        self.transform(new)
```
Comment (ksunden) [inline WrightTools/data/_data.py]: Maintain units?

Comment (untzag) [reply WrightTools/data/_data.py]: done

File: WrightTools/_base.py [hunk 3]
```diff
@@ -191,6 +209,9 @@ def __version__(self):
             self.file.attrs['__version__'] = wt5_version
         return self.file.attrs['__version__']
 
+    def _update(self):
+        pass
```
Comment (ksunden) [inline WrightTools/_base.py]: Does this serve a purpose?

File: WrightTools/_base.py [hunk 4]
```diff
@@ -245,3 +266,36 @@ def close(self):
...
+
+        The name will be set to str(val), and its natural naming identifier
+        will be wt.kit.string2identifier(str(val))
+
+        Keyword Arguments
+        -----------------
+        Each argument should have the key of a current axis or channel,
+            and a value which is a string of its new name.
+        """
+        # TODO: what happens to lists channel_names, variable_names etc?
```
Comment (ksunden) [inline WrightTools/_base.py]: Still todo?

File: WrightTools/_base.py (hunk 3 above)
Comment (untzag) [reply WrightTools/_base.py]: just as a placeholder

eventually want to migrate a bunch of natural naming code here

**Pull request merged**